middleware/near.js

```diff
@@ -74,19 +105,11 @@ function createCheckAccountDoesNotExistMiddleware({ source, fieldName }) {
             accountId = ctx.params[fieldName];
         }
 
-        // TODO: near-api-js should have explicit accoutn existence check
-        let remoteAccount = null;
-        try {
-            remoteAccount = await (await ctx.near.account(accountId)).state();
-        } catch (e) {
-            if (e.type === 'AccountDoesNotExist') {
-                return await next();
-            }
-            throw e;
-        }
-        if (remoteAccount) {
```

Comment:
Removed this check as we should always have an `account` at this point, `near.account().state()` would throw otherwise - correct me if I am wrong though.